cypress/e2e/rich-text/RichTextEditor.Tracking.spec.ts

```diff
@@ -610,8 +608,9 @@ describe('Rich Text Editor - Tracking', { viewportHeight: 2000 }, () => {
           // Part 3:
           // Update entry link to asset link
 
-          richText.editor.findByTestId('cf-ui-text-link').click({ force: true });
+          richText.editor.findByTestId('cf-ui-text-link');
```

Comment:
any expections/assertions here? same for Line 632

Reply:
ahh missed those, we can remove them 👍